spec/rails_helper.rb

```diff
@@ -4,11 +4,39 @@
 require "rspec/rails"
 require "capybara/rails"
 require "helpers/helpers_spec.rb"
+#require 'capybara/poltergeist'
+require 'phantomjs/poltergeist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.